janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/compat/AbstractESCompat.java

```diff
@@ -72,9 +74,16 @@ public String scriptLang() {
         return "painless";
     }
 
+    public ImmutableMap.Builder prepareScript(String inline, List<Map<String, Object>> fields) {
+        Map<String, Object> script = ImmutableMap.of(ES_SOURCE_KEY, inline,
+            ES_PARAMS_KEY, ImmutableMap.of(ES_PARAMS_FIELDS_KEY, fields),
+            ES_LANG_KEY, scriptLang());
+        return ImmutableMap.<String, Object>builder().put(ES_SCRIPT_KEY, script);
+    }
+
     public ImmutableMap.Builder prepareScript(String inline) {
```

Comment:
Here also, I think we can rename `inline` argument to `source`

Reply:
done